test/test-lambda.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const sinon = require('sinon');
```

Comment:
This is great stuff 🚀It'd be great if we could open another PR only with this sinon integration, just to keep the two independent. No worries if it's too much trouble, but i'd appreciate it.

Reply:
I was having issues with the testing.
So I started using sinon and then it took over.
Will it be possible to rather start a now request for testing improvements?

Reply:
No worries then, I understand. Let's keep it here. Then yes, let's create a new issue & PR for testing improvements once this is merged.

Reply:
If all other changes are good.
Give me a day or two and maybe I can revert the tests and then create a new PR for the tests.
Added a simple why to suppress the logs during tests runs.

Reply:
Ok was easier than I thought.
Reverted the changes.
Will create a separate PR with the test stuff.